eb_hooks.py

```diff
@@ -258,6 +258,10 @@ def pre_configure_hook_wrf_aarch64(self, *args, **kwargs):
     else:
         raise EasyBuildError("WRF-specific hook triggered for non-WRF easyconfig?!")
 
+def pre_test_hook_ignore_failing_tests_SciPybundle(self, *args, **kwargs):
+    if self.name == 'SciPy-bundle' and '/neoverse_v1/' in self.installdir:
```

Comment:
@laraPPr looks good, but I would add some context here by including a comment.

Maybe the condition should also be a bit more specific for now, since installing `SciPy-bundle/2021.05-foss-2021a` worked fine (because we weren't so strict on the numpy/scipy tests yet then).

We want to restrict this to only ignore the failing tests on `neoverse_v1`, but the check for `/neoverse_v1` in `self.installdir` is a bit of a hack, I would check the value of `${EESSI_SOFTWARE_SUBDIR}` instead (which is used by `configure_easybuild` to set `$EASYBUILD_INSTALLPATH`).

So, something like this (untested, but should work):

```python
def pre_test_hook_ignore_failing_tests_SciPybundle(self, *args, **kwargs):
    # ignore failing test suite for numpy & co when installing SciPy-bundle on Arm neoverse_v1,
    # see <link to EESSI/software-layer issue with more goes here>
    cpu_target = get_eessi_envvar('EESSI_SOFTWARE_SUBDIR')
    if self.name == 'SciPy-bundle' and self.version == '2021.10' and cpu_target == 'aarch64/neoverse_v1':
```

Note that `get_eessi_envvar` will fail if `${EESSI_SOFTWARE_SUBDIR}` is not set, which is a good thing (in case these hooks are used in a different context).